submissions/dikun/frogger-game/app.js

```diff
@@ -0,0 +1,113 @@
+"use srtict";
+
+const CELL = {
+	width: 100,
+	height: 85
+};
+
+const GAME_FIELD = {
+	top: 0,
+	right: CELL.width * 5,
+	bottom: CELL.height * 5,
+	left: 0
+};
+
+const PLAYER_START = {
+	x: 202,
+	y: 400
+};
+
+const ENEMY_SPEED = {
+	min: 90,
+	max: 250 
+}; 
+
+const SPRITE = {
+	enemy: 'images/enemy-bug.png',
+	player: 'images/char-boy.png'
+};
+
+const SPEED = () => ENEMY_SPEED.min + Math.floor(Math.random() * ENEMY_SPEED.max);
+const OVERLAP_AMOUNT = 70;
+const BORDER = 20;
+
+const Enemy = function (x, y, SPEED, player) {
```

Comment:
Function parameters (e.g. `SPEED`) are not constants anymore. We may send constants to functions, but the function itself doesn't really care where the parameter value comes from.